open_graph.py

```diff
@@ -29,9 +30,13 @@ def tag_article(instance):
     ogtags = [('og:title', instance.title),
               ('og:type', 'article')]
 
+    soup = BeautifulSoup(instance._content, 'html.parser')
+    img_links = soup.find_all('img')
     image = instance.metadata.get('og_image', '')
     if image:
         ogtags.append(('og:image', image))
+    elif (len(img_links) > 0):
+        ogtags.append(('og:image', img_links[0].get('src')))
 
```

Comment:
Fair enough.

settings['OUTPUT_PATH'] does not provide the right prefix though. I had a look at the structures, and I'm not finding the correct prefix for an absolute path.